src/entity/user/UserStreakAction.ts

```diff
@@ -18,7 +18,7 @@ export class UserStreakAction {
   @PrimaryColumn({ type: 'text' })
   type: UserStreakActionType;
 
-  @PrimaryColumn({ type: 'timestamptz', default: () => 'now()' })
+  @PrimaryColumn({ type: 'timestamp', default: () => 'now()' })
```

Comment:
More of a request if we can safely modify this and it will not change stored DB values.
Also can we do it for all other `timestamptz` fields?

Reply:
There is currently no record for this table, so in this instance, we are safe.

Reply:
For the other fields, we should maybe create a separate ticket, so further investigation can be done and the change must be careful since it involves data changes. Wdyt?

Reply:
Yeah wasn't aware this was just a new table :)  (empty)
so yeah that makes total sense then.